BioPage.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Peronal Portfolio Website</title>
+    <link rel="stylesheet" href="styles.css"/>
+</head>
+<body>
+    <div class="main">
+        <nav> 
+            <ul>
+                <li><a href="HomePage.html">HOME</a></li>
+                <li><a href="BioPage.html">ABOUT ME</a></li>
+                <li><a href="ExperiencePage.html">EXPERIENCE PAGE</a></li>
+                <li><a href="PortfolioPage.html">PORTFOLIO PAGE</a></li>
+            </ul>
+        </nav>
+        <div class="homeInfo">
+            <h1><span>About Me</span></h1>
+            <p>Born and raised in Georgia (Savannah & Atlanta). I've always had a passion for technlogy.
+                <br>I have been playing video game since I was little, and I always wanted to make one of
+                <br>my own. From there I self-studied Java and Python programming. After a bit, I came to
+                <br>out that it would be more beneficial to know how a computer works before I can tell it
+                <br>what to do. I began taking my passion for I.T more seriously. I purchased CompTIA A+ 
+                <br>books and training guides to prep myself for the CompTIA A+ exam. At the same time,
+                <br>I gained hands on experience by helping friends and family with technical issues. I 
+                <br>began to become more tech savvy as I learned company operations, processes, and advanced
+                <br>my technical computer skills even further. As my experience grew, my opportunities did
+                <br>as well. I went from being help-desk support to being an I.T Administrator - resolving
+                <br>tickets and issues globally for companies. After completing the CompTIA A+ exam, I began 
+                <br>studying for my CompTIA Network+ certification. Currently I am on the brink of passing my
+                <br>CompTIA Network+ exam and then I will be onto CompTIA Security+ or pick up my Java or Python
+                <br>programming studies. I love everything about technology. I have a passion for creating and
+                <br>helping people. Being able to bring my skills to the table and still be creative whilst doing
+                <br>so. Whether I am rebuilding or repairing computers, remote support, or networking technicalit-
+                <br>ies. I enjoy doing every bit of the job. Learning new technologies, company processes, and 
+                <br>meeting great people who have the same passion!
+            </p> 
```

Comment:
some of this text overlaps with the footer when the font size is zoomed in enough